services/sr_asset_assign.py

```diff
@@ -153,19 +169,22 @@ def main(args):
     container = args.container
     logger.info(args)
 
-    # Get AGOL Token
-    token_agol = create_agol_login_token()
-
     # Getting SR data from Knack
     config = CONFIG[app_name][container]
     modified_date_field = config["modified_date_field"]
     kwargs = {"scene": config["scene"], "view": args.container}
     data = knackpy.api.get(app_id=APP_ID, api_key=API_KEY, **kwargs)
 
+    # ignore records that have a null location data
+    data = [d for d in data if d[config["x_field"]] and d[config["y_field"]]]
+
```

Comment:
moving this check above the, `No SRs waiting in queue to be processed, doing nothing.` logging. Currently there was a CSR with no location just sitting there which prevents the proper logging message from appearing.